src/domains/ecommerce-app/04_apim_ecommerce.tf

```diff
@@ -313,3 +313,114 @@ module "apim_pagopa_notifications_service_api_v1" {
     hostname = local.ecommerce_hostname
   })
 }
+
+##############################
+## API helpdesk service ##
+##############################
+locals {
+  apim_pagopa_helpdesk_service_api = {
+    display_name          = "pagoPA ecommerce - helpdesk service API"
+    description           = "API to support helpdesk service"
+    path                  = "ecommerce/helpdesk-service"
+    subscription_required = true
+    service_url           = null
+  }
+}
+
+# helpdesk service APIs
+resource "azurerm_api_management_api_version_set" "pagopa_helpdesk_service_api" {
+  name                = format("%s-helpdesk-service-api", local.project)
+  resource_group_name = local.pagopa_apim_rg
+  api_management_name = local.pagopa_apim_name
+  display_name        = local.apim_pagopa_helpdesk_service_api.display_name
+  versioning_scheme   = "Segment"
+}
+
+
+#helpdesk assistance api for ecommerce
+module "apim_pagopa_helpdesk_service_api_v1" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_api?ref=v6.6.0"
+
+  name                  = format("%s-helpdesk-service-api", local.project)
+  api_management_name   = local.pagopa_apim_name
+  resource_group_name   = local.pagopa_apim_rg
+  product_ids           = [module.apim_ecommerce_product.product_id]
+  subscription_required = local.apim_pagopa_helpdesk_service_api.subscription_required
+  version_set_id        = azurerm_api_management_api_version_set.pagopa_helpdesk_service_api.id
+  api_version           = "v1"
+
+  description  = local.apim_pagopa_helpdesk_service_api.description
+  display_name = local.apim_pagopa_helpdesk_service_api.display_name
+  path         = local.apim_pagopa_helpdesk_service_api.path
+  protocols    = ["https"]
+  service_url  = local.apim_pagopa_helpdesk_service_api.service_url
+
+  content_format = "openapi"
+  content_value = templatefile("./api/ecommerce-helpdesk-api/v1/_openapi.json.tpl", {
+    hostname = local.apim_hostname
+  })
+
+  xml_content = templatefile("./api/ecommerce-helpdesk-api/v1/_base_policy.xml.tpl", {
+    hostname = local.ecommerce_hostname
+  })
+}
+
+
+##############################
+## API helpdesk assistance service ##
```

Comment:
`helpdesk` and `assistance` are synonyms, what do you think of `technical-helpdesk`? in order to refer also to the client/product

Reply:
cc @Fernando-Granato